src/components/controlButtons/controlButtons.js

```diff
@@ -0,0 +1,52 @@
+import { createElement } from "../../utils/elements";
+
+export const ControlButtons = () => {
+  const OuterDIV = createElement("div", {
+    className: "nokiaControlButtons",
+  });
+
+  const numbersFirstRow = [{ name: "1" }, { name: "2" }, { name: "3" }];
+  const numbersSecondRow = [{ name: "4" }, { name: "5" }, { name: "6" }];
+  const numbersThirdRow = [{ name: "7" }, { name: "8" }, { name: "9" }];
+  const numbersFourthRow = [{ name: "*" }, { name: "0" }, { name: "#" }];
+
+  const firstRow = createElement("div");
+  const secondRow = createElement("div");
+  const thirdRow = createElement("div");
+  const fourthRow = createElement("div");
```

Comment:
It would be more performant if you would only use one array for all buttons, so it would only take one forEach loop and only one div for containing the buttons.
You could set the width of the buttons to 33% and use flexbox with wrap.

Reply:
thank you for your advice.
this was the logic problem we had and after trying it with one array, we decided to resolve it this way, because of getting out of time -> your advice is the solution we searched for.